### PR TITLE
doc: add feature categorizations

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## What this PR changes/adds

this document should help contributors to decide how and where their code should be hosted

## Why it does that

There is a need from the community to have guidance on that topic.

## Checklist

- [ ] added appropriate tests?
- [ ] performed checkstyle check locally?
- [ ] added/updated copyright headers?
- [ ] documented public classes/methods?
- [ ] added/updated relevant documentation?
- [ ] added relevant details to the changelog? (_skip with label `no-changelog`_)
- [x] formatted title correctly? (_take a look at the [CONTRIBUTING](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CONTRIBUTING.md#submit-a-pull-request) and [styleguide](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/styleguide.md) for details_)

### CODE REVIEW

**Review by juliapampus — APPROVED** [review]
> Get referenced as "friend"

I assume this means a repo outside our org. Does 

> Get adopted in EDC core

covers both either a separate repo in the edc org, or an "extension" in the core repo? Or do we need to distinguish further?

**Comment by paullatzelsperger** [comment]
> > Get referenced as "friend"
> 
> I assume this means a repo outside our org. Does
> 
> > Get adopted in EDC core
> 
> covers both either a separate repo in the edc org, or an "extension" in the core repo? Or do we need to distinguish further?

incorporated both suggestions.

File: contribution_categories.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+In order to become a "friend" of EDC, we do a quick scan of the code base to make sure it does not contain anything offensive, or that contradicts our code of conduct, ethics or other core OSS values. 
+
+The EDC core team does not maintain or endorse "friend" projects in any way, nor is it responsible for it, but we do provide a URL list to make it easier for other developers to get an overview of the EDC market spread.
+
+This is the easiest way to "get in" and will be the applicable form of adoption for _most_ features and projects.
+
+## Get adopted in EDC core
+
+This means the contribution gets added to the EDC code base, and is henceforth maintained by the EDC core team. The barrier of entry for this is much higher than for "friends", and a more in-depth review of the code will be performed.
+Note that the
```
Comment (juliapampus) [inline contribution_categories.md:21]: I guess something is missing here?

**Review by juliapampus — CHANGES_REQUESTED** [review]

File: contribution_categories.md (hunk 1 above)
Comment (paullatzelsperger) [reply contribution_categories.md]: there sure is ^^

**Review by juliapampus — APPROVED** [review]

**Comment by juliapampus** [comment]
I am about to extend the issue templates. Will open up a PR soon.

**Pull request merged**